dictionaries/CallMap.php

```diff
@@ -7878,12 +7878,12 @@
 'mysqli_execute' => ['bool', 'statement'=>'mysqli_stmt', 'params='=>'list<mixed>|null'],
 'mysqli_execute_query' => ['mysqli_result|bool', 'mysql'=>'mysqli', 'query'=>'non-empty-string', 'params='=>'list<mixed>|null'],
 'mysqli_fetch_all' => ['list<array<array-key,null|int|float|string>>', 'result'=>'mysqli_result', 'mode='=>'3'],
-'mysqli_fetch_all\'1' => ['list<array<string,null|int|float|string>>', 'result'=>'mysqli_result', 'mode='=>'1'],
```

Comment:
I think this can just be `array<...>` instead of having to use array-key.

Reply:
Wasn't sure if its better to be explicit here, and there were a handful of other entries already using array-key.

Both should work.

Reply:
Up to the maintainers, but I think the distinction is use array-key outside of an array but it's not necessary in an array.